spring-kafka/src/main/java/org/springframework/kafka/listener/ConsumerSeekAware.java

```diff
@@ -68,6 +69,27 @@
 		 */
 		void seek(String topic, int partition, long offset);
 
+		/**
+		 * Queue a seekToBeginning operation to the consumer. The seek will occur after
+		 * any pending offset commits. The consumer must be currently assigned the
+		 * specified partition.
+		 * @param topic the topic.
+		 * @param partition the partition.
+		 */
+		default void seekToBeginning(String topic, int partition) {
+			seek(topic, partition, 0);
```

Comment:
I'm not sure in such a default implementation.
It doesn't reflect reality about `BEGINNING` and there is no intentions to expose this interface for end-users.
This is just an internal Framework hook to let end-users to perform seeks on the provided callback.
No body is going to implement it himself.

I will be pretty happy do not have this interface as a `@FunctionalInterface`.
Just IMHO...